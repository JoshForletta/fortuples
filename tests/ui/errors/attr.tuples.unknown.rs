use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::unknown]
    impl Container<#Tuple> {}
}

fn main() {}
