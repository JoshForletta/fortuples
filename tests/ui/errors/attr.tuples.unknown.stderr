error: unknown setting
 --> tests/ui/errors/attr.tuples.unknown.rs:6:15
  |
6 |     #[tuples::unknown]
  |               ^^^^^^^
